src/qml/components/AttachmentPreview.qml

```diff
@@ -1,16 +1,21 @@
 import QtQuick 2.12
 import QtQuick.Controls 2.12
 
+import "../theme"
+
 Item {
```

Comment:
You are wrapping into Item again. When user opens preview in messenger he sees Image inside Rectangle, why not use this simple hierarchy?

Reply:
Not me.